lib/internal/url.js

```diff
@@ -930,6 +957,30 @@ class URL {
   toJSON() {
     return this[kFormat]({});
   }
+
+  static createObjectURL(obj) {
+    const blob = lazyBlob();
+    if (!blob.isBlob(obj))
+      throw new ERR_INVALID_ARG_TYPE('obj', 'Blob', obj);
+
+    const id = lazyCryptoRandom().randomUUID();
+
+    storeDataObject(id, obj[blob.kHandle], obj.size, obj.type);
+
+    return `blob:node:${lazyThreadId()}:${id}`;
+  }
+
+  static revokeObjectURL(url) {
+    url = `${url}`;
+    try {
+      const parsed = new URL(url);
+      const { 2: id } = parsed.pathname.split(':');
+      if (id !== undefined)
+        revokeDataObject(id);
```

Comment:
nit: we can protect in case of `Object.defineProperty(Array.prototype, '2', { get: () => 'something' })`
```suggestion
      const split = StringPrototypeSplit(parsed.pathname, ':', 3);
      if (split.length === 3)
        revokeDataObject(split[2]);
```